graphql/database/missions/__tests__/utils.test.js

```diff
@@ -0,0 +1,313 @@
+  started: '2017-07-19T03:05:12Z',
+  tabGoal: 1000,
+  acceptedSquadMembers: ['cL5KcFKHd9fEU5C9Vstj3g4JAc73', 'abcdefghijklmno'],
+  pendingSquadMembersExisting: [],
+  pendingSquadMembersEmailInvite: [],
+  rejectedSquadMembers: [],
+  endOfMissionAwards: [],
+  missionType: 'cats',
+}
+
+const mockUsers = [
+  {
+    id: 'abcdefghijklmno',
+    email: 'kevin@example.com',
+    username: 'kevin',
+    joined: '2017-07-18T20:45:53Z',
+    vcCurrent: 0,
+    vcAllTime: 0,
+    level: 1,
+    tabs: 1,
+    validTabs: 0,
+    maxTabsDay: {
+      maxDay: {
+        date: '2017-07-18T20:45:53Z',
+        numTabs: 1,
+      },
+      recentDay: {
+        date: '2017-07-18T20:45:53Z',
+        numTabs: 1,
+      },
+    },
+    heartsUntilNextLevel: 5,
+    vcDonatedAllTime: 0,
+    numUsersRecruited: 3,
+    backgroundImage: {
+      id: 'fb5082cc-151a-4a9a-9289-06906670fd4e',
+      timestamp: '2019-02-22T20:09:19.543Z',
+      image: 'lake.jpg',
+      thumbnail: 'lake.jpg',
+    },
+    backgroundOption: 'photo',
+    backgroundColor: null,
+    customImage: null,
+    lastTabTimestamp: '2017-07-18T20:45:53Z',
+    created: '2017-07-18T20:45:53Z',
+    updated: '2017-07-18T20:45:53Z',
+  },
+  {
+    id: 'cL5KcFKHd9fEU5C9Vstj3g4JAc73',
+    email: 'alec+01@tabforacause.org',
+    username: 'alec',
+    v4BetaEnabled: true,
+    currentMissionId: '123456789',
+    joined: '2017-07-19T03:05:12Z',
+    vcCurrent: 50,
+    vcAllTime: 100,
+    level: 4,
+    tabs: 130,
+    validTabs: 100,
+    maxTabsDay: {
+      maxDay: {
+        date: '2017-07-19T03:05:12Z',
+        numTabs: 130,
+      },
+      recentDay: {
+        date: '2017-07-19T03:05:12Z',
+        numTabs: 130,
+      },
+    },
+    heartsUntilNextLevel: 25,
+    vcDonatedAllTime: 50,
+    numUsersRecruited: 0,
+    backgroundImage: {
+      id: '9308b921-44c7-4b4e-845d-3b01fa73fa2b',
+      image: '94bbd29b17fe4fa3b45777281a392f21.jpg',
+      thumbnail: '5d4dfd0b34134879903f0480720bd746.jpg',
+      timestamp: '2017-07-19T03:05:12Z',
+    },
+    backgroundOption: 'photo',
+    backgroundColor: null,
+    customImage: null,
+    lastTabTimestamp: '2017-07-19T16:22:59Z',
+    created: '2017-07-19T03:05:12Z',
+    updated: '2017-07-19T16:22:59Z',
+  },
+  {
+    id: 'efghijklmnopqrs',
+    email: 'sandra@example.com',
+    username: 'sandra',
+  },
+]
+const mockDefaultMissionReturn = {
+  missionId: '123456789',
+  status: 'started',
+  squadName: 'TestSquad',
+  tabGoal: 1000,
+  endOfMissionAwards: [],
+  created: '2017-07-19T03:05:12Z',
+  tabCount: 258,
+  squadMembers: [
+    {
+      username: 'alec',
+      status: 'accepted',
+      longestTabStreak: 4,
+      currentTabStreak: 2,
+      missionMaxTabsDay: 10,
+      tabs: 234,
+    },
+    {
+      username: 'kevin',
+      status: 'accepted',
+      longestTabStreak: 4,
+      currentTabStreak: 2,
+      missionMaxTabsDay: 10,
+      tabs: 24,
+    },
+  ],
+}
+
+describe('buildMissionReturnType tests', () => {
```

Comment:
For all async tests, add `expect.assertions(1)` at the top. Otherwise, there are cases where test could exit successfully without actually calling some assertions.